pkg/util/helm/bundle.go

```diff
@@ -0,0 +1,75 @@
+package helm
+
+import (
+	"fmt"
+
+	kudo "github.com/kudobuilder/kudo/pkg/apis/kudo/v1alpha1"
+	"github.com/kudobuilder/kudo/pkg/bundle"
+
+	"github.com/helm/helm/pkg/chartutil"
+)
+
+// ToBundle converts the helm chart on disk into a Kudo Bundle
```

Comment:
we should really use one naming. The thing we have are bundles or packages? we are not very consistent on this. I don't really care but we should have one name :)